Makefile

```diff
@@ -1,5 +1,5 @@
 REGISTRY_NAME?=docker.io/sbezverk
-IMAGE_VERSION?=0.0.0
+IMAGE_VERSION?=test-235
```

Comment:
this I will revert (used only for temp testing to make the container)

Reply:
reverted in https://github.com/sbezverk/gobmp/pull/236/commits/43f15de7a282824ce0fbc241654ca299fb6dbe40